tests/src/common/Wsk.scala

```diff
@@ -149,9 +149,14 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
      */
     def get(
         name: String,
+        summary: Boolean = false,
         expectedExitCode: Int = SUCCESS_EXIT)(
             implicit wp: WskProps): RunResult = {
-        cli(wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name)), expectedExitCode)
+        var cliparms = wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name))
+        if (summary) {
+            cliparms = cliparms ++ Seq("--summary")
+        }
+        cli(cliparms, expectedExitCode)
```

Comment:
@markusthoemmes - Very good point.  I did choose to use `var` here so that there was a single `cli` execution line allowing for the easy addition of other flags in the future.